Formula/pymol.rb

```diff
@@ -70,7 +70,7 @@ def install
 
     ENV.append "CPPFLAGS", "-I#{Formula["freetype"].opt_include}"
 
-    # Note: openvr support is not included.
+    # openvr support is not included.
```

Comment:
```suggestion
    # Note: openvr support is not included.
```
Please revert this unrelated change.

Reply:
For some reason, this change is necessary to pass the CI. Here is the message:

```
==> brew doctor

==> Running TapSyntax#run!
==> brew readall --aliases brewsci/bio
==> brew style brewsci/bio
==> FAILED
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/brewsci/homebrew-bio/Formula/pymol.rb:73:7: C: Annotation keywords like Note should be all upper case, followed by a colon, and a space, then a note describing the problem.
    # Note: openvr support is not included.
      ^^^^^^

305 files inspected, 1 offense detected, 1 offense auto-correctable
```

I've never encountered this message until today.

Reply:
Me neither. That's a new one to me. I've pushed 8b30424cbe14bb2c089e20f2580e4d33b10fb696 to remove `Note:`.

Reply:
You can rebase this PR onto `origin/develop` now.

Reply:
Thank you. This PR is ready to merge now.